client/test/e2e/playwright/auth.setup.ts

```diff
@@ -0,0 +1,21 @@
+import { test as setup, expect } from '@playwright/test';
+
+const authFile = 'playwright/.auth/user.json';
+
+setup('authenticate', async ({ page }) => {
+  // Perform authentication steps for authentication process.
+  await page.goto('./');
+  await page
+    .getByRole('button', { name: 'GitLab logo Sign In with GitLab' })
+    .click();
+  await page.getByRole('button', { name: 'Accept All Cookies' }).click();
+  await page.fill('#user_login', 'TestUserDTaaS'); // Insert valid GitLab testing username.
```

Comment:
please bring these out to env file of test/ directory. You can make it into .env and load it via [dotnev](https://github.com/motdotla/dotenv) package.